docs/content/latest/explore/backup-restore/point-in-time-recovery-ysql.md

```diff
@@ -0,0 +1,214 @@
+                    "interval": "60.000s",
+                    "retention": "600.000s"
+                },
+                "snapshots": [
+                    {
+                        "id": "8d588cb7-13f2-4bda-b584-e9be47a144c5",
+                        "snapshot_time_utc": "2021-05-07T20:16:08.492330+0000"
+                    }
+                ]
+            }
+        ]
+    }
+    ```
+
+### Restore from an absolute time
+
+1. From a command prompt, get a timestamp.
+
+    ```sh
+    $ python -c 'import datetime; print datetime.datetime.now().strftime("%s%f")'
+    ```
+
+    ```output
+    1620418817729963
+    ```
+
+1. Add a row for employee 9999 to the table:
+
+    ```sql
+    INSERT INTO employees (employee_no, name, department, salary) 
+      VALUES 
+      (9999, 'Wrong Name', 'Marketing', 10000);
+
+    SELECT * FROM employees;
+    ```
+
+    ```output
+     employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            9999 | Wrong Name     | Marketing  |  10000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (5 rows)
+    ```
+
+1. Restore the snapshot schedule to the timestamp you obtained before you added the data, at a terminal prompt.
+
+    ```sh
+    $ bin/yb-admin restore_snapshot_schedule 0e4ceb83-fe3d-43da-83c3-013a8ef592ca 1617670679185100
+    ```
+
+    ```output
+    {
+        "snapshot_id": "2287921b-1cf9-4bbc-ad38-e309f86f72e9",
+        "restoration_id": "1c5ef7c3-a33a-46b5-a64e-3fa0c72709eb"
+    }
+    ```
+
+1. Next, verify the restoration is in `RESTORED` state (you'll see more snapshots in the list, as well):
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    8d588cb7-13f2-4bda-b584-e9be47a144c5  COMPLETE
+    1f4db0e2-0706-45db-b157-e577702a648a  COMPLETE
+    b91c734b-5c57-4276-851e-f982bee73322  COMPLETE
+    04fc6f05-8775-4b43-afbd-7a11266da110  COMPLETE
+    e7bc7b48-351b-4713-b46b-dd3c9c028a79  COMPLETE
+    2287921b-1cf9-4bbc-ad38-e309f86f72e9  COMPLETE
+    97aa2968-6b56-40ce-b2c5-87d2e54e9786  COMPLETE
+    Restoration UUID                      State
+    1c5ef7c3-a33a-46b5-a64e-3fa0c72709eb  RESTORED
+    ```
+
+1. In the YSQL shell, verify the data is restored, without a row for employee 9999:
+
+    ```sql
+    yugabyte=# select * from employees;
+    ```
+
+    ```output
+     employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (4 rows)
+    ```
+
+### Restore from a relative time
```

Comment:
this seems simple enough as a note, maybe we can have the same thing for YCQL -- nice and compact

Reply:
I like this idea. I've truncated the YCQL version, and replicated it here.